src/gateway.php

```diff
@@ -403,6 +403,9 @@ public function thank_you_page( $order_id ) {
 			<p>
 				<?php echo esc_html( $description ); ?>
 			</p>
+			<p>
```

Comment:
Might be worth adding a CSS class here so people can hide this block with CSS if they want?